test.sh

```diff
@@ -110,7 +110,7 @@ vagrantDestroy
 nodesOff
 
 # Power on vagrant box and nodes 
-vagrantUp
+vagrantUp "RackHD/packer/ansible/roles/monorail/files/auth_config"
```

Comment:
For split NB/SB, the previous config.json has already do that way. And integration test works with NB/SB (no auth) config for a period of time. The only difference this time is add an additional NB endpoint with auth enabled. And new added auth test will running against that new endpoint. So I don't think it will break the existing test.